Gruntfile.coffee

```diff
@@ -1169,6 +1198,20 @@ module.exports = (grunt) ->
     'exec:androidReplaceXwalkDist'
   ]
 
+  # Identical to the release build but with monkey-proofing, for use by
+  # automated test systems without contaminating usage metrics.
+  grunt.registerTask 'monkey_proof_android', [
```

Comment:
OK.
 1. Would you like to remove this from all dev builds, or just for Android?
 1. Would you like to remove the opt-in screen, or just make it ineffective?
 1. Would you like to trigger this by removing the metrics provider from freedom-module.json, or in some other way?